src/pyedb/dotnet/edb_core/edb_data/padstacks_data.py

```diff
@@ -814,7 +814,7 @@ def convert_to_3d_microvias(self, convert_only_signal_vias=True, hole_wall_angle
         except KeyError:  # pragma: no cover
             start_elevation = layers[self.instances[0].start_layer].lower_elevation
         try:
-            stop_elevation = layers[self.via_start_layer].upper_elevation
+            stop_elevation = layers[self.via_stop_layer].upper_elevation
```

Comment:
@svandenb-dev do you also have time to remove the usage of try? It make it difficult to debug when something goes wrong.

Reply:
@ring630 done